src/components/App/App.component.jsx

```diff
@@ -4,51 +4,38 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
 import LoginPage from '../../pages/Login';
+import Favorites from '../../pages/Favorites';
 import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
+import Sidebar from '../Sidebar';
 import Private from '../Private';
-import Fortune from '../Fortune';
+// import Videos from '../Videos';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import Navbar from '../Navbar';
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
-
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
-
-    const intervalId = setInterval(rotateBackground, 3000);
-    body.addEventListener('click', rotateBackground);
-
-    return () => {
-      clearInterval(intervalId);
-      body.removeEventListener('click', rotateBackground);
-    };
-  }, []);
+  useLayoutEffect(() => {}, []);
```

Comment:
What is the use for this line? If it does not do anything, it should be removed from the code